omod/src/main/webapp/pages/patient/encounterList.gsp

```diff
@@ -1,6 +1,15 @@
 <%
     ui.decorateWith("appui", "standardEmrPage")
     ui.includeJavascript("uicommons", "datatables/jquery.dataTables.min.js")
+
+    // hack, hardcode what encounter types use the simple view
```

Comment:
Obviously a hack... we could consider way to make this configurable, but I wasn't sure if it was worth the effort, because it seemed like it would just be a hacky way to make it configurable (ie gp: "pihcore.encounterTypesRenderedWithSimpleView")?  At least here it's hack, but not propagated and use is limited to a single file.